src/editor/Gui.cpp.bak

```diff
@@ -0,0 +1,3238 @@
+#include "Gui.h"
```

Comment:
can delete this file. Git is the backup :>

Reply:
Yes sorry, it created after WinMerge )